Unix/configure

```diff
@@ -154,6 +155,9 @@ do
       certsdir=$arg 
       ;;
 
+    --credsdir=*)
+      credsdir=$arg 
+      ;;
```

Comment:
If the NTLM file is coming out of the omiserver.conf file, then why does this flag still exist, and what does it control?

Also, my prior comment was disregarded. Please fix formatting if this does belong (and explain why so I understand). Still looking for a blank line after 160 like everything around it has. Code should be consistently formatted.